src/jose/jws.ts

```diff
@@ -1,9 +1,20 @@
 import * as ed25519 from './algorithms/ed25519';
-import * as jwk from './jwk';
 import * as secp256k1 from './algorithms/secp256k1';
-import base64url from 'base64url';
+
+import { base64url } from 'multiformats/bases/base64';
+
 import type { JwkPrivate, JwkPublic } from './jwk';
 
+const verifiers = {
```

Comment:
```suggestion
const verifiers: { [key:string]: (payload: Uint8Array, signature: Uint8Array, publicKeyJwk: JwkPublic) => Promise<boolean> } = {
```
And remove type declaration in line 65.

Consider the same thing for `signers` map

Reply:
✅